andela/gradr
# Ft implement monaco editor 166702872

#### What does this PR do?
Switches playground editor from Code Mirror Editor to Monaco editor

#### Description of Task to be completed
- add monaco-editor module
- initialise Monaco editor in `src/commons/js/monaco-init.js`
- add Monaco language file in `src/commons/js/monaco-lang.js`
- update `src/playground/js/playground.js`
- update `src/dash/js/spec.js`

#### How should this be manually tested?
- switch to the branch `ft-implement-monaco-editor-166702872`
- start the server using `yarn develop-playground`
- navigate to `http://localhost:1234/jqe3zYO8xTfiuCLDEEgu` on the browser
- you will notice the Monaco editor with prepopulated codes

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#166702872](https://www.pivotaltracker.com/story/show/166702872)
[#166771642](https://www.pivotaltracker.com/story/show/166771642)

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/25525765/61220100-e59e0100-a715-11e9-9749-39f5254b65f6.png)
![image](https://user-images.githubusercontent.com/25525765/61999631-feb79200-b0c5-11e9-8ca6-02120b6fee14.png)
![image](https://user-images.githubusercontent.com/25525765/62099467-67ea0180-b28e-11e9-8e1c-13dc6eb04a91.png)
![image](https://user-images.githubusercontent.com/25525765/62099475-7506f080-b28e-11e9-9b1e-d7aa998b63c7.png)
